.netlify/build-and-move.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Build the static web site in website/build
+# bundle install     should be cached???
```

Comment:
Looks like you got to the bottom of this in finding https://github.com/netlify/build-image/issues/196

Can you update the comment to speak to this netlify issue?